test/unit/builds_method_signature_test.rb

```diff
@@ -0,0 +1,102 @@
+require "test_helper"
+
+module Mocktail
+  class BuildsMethodSignatureTest < Minitest::Test
+    def setup
+      @subject = BuildsMethodSignature.new
+    end
+
+    def test_basic_call
+      assert_equal @subject.build(signature), "()"
+    end
+
+    def test_positional_call
+      assert_equal @subject.build(signature(
+        positional_params: Params.new(all: [:a, :b], required: [:a, :b]),
+      )),
+     "(a, b)"
```

Comment:
Most of Minitest's assertion methods like `assert_equal` are `(expected, actual)`, so I'd flip the order on each of these and coincidentally spare ourselves the kinda goofy line that starts with a string literal, like this:

```suggestion
      assert_equal "(a, b)", @subject.build(signature(
        positional_params: Params.new(all: [:a, :b], required: [:a, :b]),
      ))     
```